odkx-src/sync-endpoint-cloud-setup.rst

```diff
@@ -577,16 +577,20 @@ Launching the ODK-X Server
   .. image:: /img/setup-azure/azure11.png
    :width: 600
 
-.. note::
-    If you are unable to log in, you may need to take the docker stack down and bring it back up again. That can be done with the following commands below:
+.. _sync-endpoint-setup-take-stack-down:
```

Comment:
This seems like it needs a header for taking the stack/swarm down and bringing it back up (currently listing just taking it down).